src/panels/media-browser/ha-bar-media-player.ts

```diff
@@ -640,13 +646,14 @@ export class BarMediaPlayer extends LitElement {
       }
 
       :host([narrow]) img {
-        max-height: 80px;
-        max-width: 80px;
+        max-height: 56px;
+        max-width: 56px;
+        padding-left: 2px;
```

Comment:
images shouldn't have padding but margin. But shouldn't we do 0px ?

Reply:
I thought I removed this

Reply:
I think you are looking at old code